Styles/styles.css

```diff
@@ -0,0 +1,233 @@
+    padding: 0;
+}
+
+body {
+  background: white;
+}
+
+header {
+  position: fixed;
+  background: #fff;
+  width: 100%;
+  height: 88px;
+}
+
+
+#logo {
+  float: left;
+  width: 100px;
+  height: auto;
+  padding: 1em 0 0 1em;
+}
+
+header ul {
+  float: right;
+  padding: 4em 2em 0 0;
+}
+
+header li {
+  display: inline;
+  padding-left: .5em;
+}
+
+a:link {
+  text-decoration: none;
+  color: #00c6a0;
+}
+
+a:visited { color: #00c6a0; }
+
+a:hover { color: #fc3;}
+
+a:active { color: black;}
+
+.current-link:visited {
+  color: black;
+}
+
+
+/*********
+*Homepage*
+**********/
+
+main {
+  clear: both;
+  width: 80%;
+  height: 100%;
+  padding-top: 88px;
+  margin: 0 auto;
+}
+
+#home-text {
+  background: white;
+  max-width: 50%;
+  padding: 8em 1em;
+  margin: 20px 0;
+  float: right;
+}
+
+#home-text h1 {
+  display: block;
+  margin: 0 0 20px;
+  padding: 0 0 20px;
+  border-bottom: solid 1px #eee;
+  font-family: "Roboto Mono", "Georgia", serif;
+}
+
+
+#home-image {
+  float: left;
+  height: 300;
+  padding: 6em 1em;
+  margin-left: 4em;
+
+}
+
+#home-image img {
+  border-radius: 50%;
+}
+
+#home-foot {
+  position: absolute;
+}
+
+/************
+*Site Footer*
+*************/
+
+
+footer {
+  position: relative;
+  bottom: 0;
+  background: #fff;
+  height: 70px;
+}
+
+footer section {
+  width: 50%;
+  margin: 0 40%;
+}
+
+footer img {
+  width: 5%;
+  height: 5%;
+  padding: 1em 1em;
+  margin: .5em 0 0 0;
+}
+
+/*************
+projects page*
+*************/
+
+.project-post {
+    display: inline-block;
+    background: #f2f2f2;
+    padding: 20px;
+    margin: 20px;
+    max-width: 40%;
+}
+
+.project-post p {
+    margin: 0 0 1em;
+}
+
+.post-head h2 {
+  font-family: "Roboto Mono", "Georgia", serif;
+}
+
+.project-head p {
+    display: block;
+    margin: 0 0 20px;
+    padding: 0 0 20px;
+    border-bottom: solid 1px #fff;
+    color: #00c6a0;
+}
+
+.project-foot img {
+  height: 40px;
+  margin: 0 45%
+}
+
+
+/**********
+*blog page*
+***********/
+
+.blog-post {
+    background:  #f2f2f2;
+    padding: 20px;
+    margin: 20px 0;
+}
+
+
+.blog-post p {
+    margin: 0 0 1em;
+}
+
+
+.blog-head p {
+    display: block;
+    margin: 0 0 20px;
+    padding: 0 0 20px;
+    border-bottom: solid 1px #fff;
+    color: #00c6a0;
+}
+
+.tags {
+  color: #fc3;
+  text-align: right;
+}
+
+
+
+/*************
+* about  page*
+*************/
+
+.about-post {
+    background: #f2f2f2;
+    padding: 20px;
+    margin: 20px;
+    max-width: 40%;
+    float: right;
+}
+
+.about-img img {
+  border-radius: 50%;
+}
+
+.about-img {
+    display: inline-block;
+    background: #f2f2f2;
+    padding: 20px;
+    margin: 1.5em 0 0 6em;
+    position: fixed;
+}
+
+
+.about-post p {
+    margin: 0 0 1em;
+}
+
+.about-head h2 {
+  display: block;
+  margin: 0 0 20px;
+  padding: 0 0 20px;
+  border-bottom: solid 1px #fff;
+  color: black;
+}
+
+.about-foot a, .about-foot img {
+  margin: 0;
+  padding: 0;
+  display: inline;
+  height: 40px;
```

Comment:
I'd recommend standardizing the sizing format (%, px, em/rem) you use to set up heights and widths. One thing I used to do when making my .css file was to take all of the classes and ids in a section and cross off duplicate lines - it helped me to see when things weren't as DRY as they could be. Having standardized sizing formats helped make it obvious when I was repeating myself unnecessarily.

You're missing H1 tags, which may cause screen readers to have a hard time orienting themselves on your site. Your .css file could use some trimming and DRYing up, but overall solid project. 👍